arbitrum-docs/useful-addresses.mdx

```diff
@@ -33,30 +33,30 @@ _Users should only interact with the token bridge via dapp interfaces like https
+| L2 ERC20 Gateway      | [0x09e9222E96E7B4AE2a407B98d48e330053351EEe](https://arbiscan.io/address/0x09e9222E96E7B4AE2a407B98d48e330053351EEe)  | [0xcF9bAb7e53DDe48A6DC4f286CB14e05298799257](https://nova.arbiscan.io/address/0xcF9bAb7e53DDe48A6DC4f286CB14e05298799257) | [0x2eC7Bc552CE8E51f098325D2FcF0d3b9d3d2A9a2](https://goerli.arbiscan.io/address/0x2eC7Bc552CE8E51f098325D2FcF0d3b9d3d2A9a2)          |
+| L1 Arb-Custom Gateway | [0xcEe284F754E854890e311e3280b767F80797180d](https://etherscan.io/address/0xcEe284F754E854890e311e3280b767F80797180d) | [0x23122da8C581AA7E0d07A36Ff1f16F799650232f](https://etherscan.io/address/0x23122da8C581AA7E0d07A36Ff1f16F799650232f)     | [0x9fDD1C4E4AA24EEc1d913FABea925594a20d43C7](https://goerli.etherscan.io/address/0x9fDD1C4E4AA24EEc1d913FABea925594a20d43C7)         |
+| L2 Arb-Custom Gateway | [0x096760F208390250649E3e8763348E783AEF5562](https://arbiscan.io/address/0x096760F208390250649E3e8763348E783AEF5562)  | [0xbf544970E6BD77b21C6492C281AB60d0770451F4](https://nova.arbiscan.io/address/0xbf544970E6BD77b21C6492C281AB60d0770451F4) | [0x8b6990830cF135318f75182487A4D7698549C717](https://goerli.arbiscan.io/address/0x8b6990830cF135318f75182487A4D7698549C717)          |
+| L1 Weth Gateway       | [0xd92023E9d9911199a6711321D1277285e6d4e2db](https://etherscan.io/address/0xd92023E9d9911199a6711321D1277285e6d4e2db) | [0xE4E2121b479017955Be0b175305B35f312330BaE](https://etherscan.io/address/0xE4E2121b479017955Be0b175305B35f312330BaE)     | [0x6e244cD02BBB8a6dbd7F626f05B2ef82151Ab502](https://goerli.etherscan.io/address/0x6e244cD02BBB8a6dbd7F626f05B2ef82151Ab502)         |
+| L2 Weth Gateway       | [0x6c411aD3E74De3E7Bd422b94A27770f5B86C623B](https://arbiscan.io/address/0x6c411aD3E74De3E7Bd422b94A27770f5B86C623B)  | [0x7626841cB6113412F9c88D3ADC720C9FAC88D9eD](https://nova.arbiscan.io/address/0x7626841cB6113412F9c88D3ADC720C9FAC88D9eD) | [0xf9F2e89c8347BD96742Cc07095dee490e64301d6](https://goerli.arbiscan.io/address/0xf9F2e89c8347BD96742Cc07095dee490e64301d6)          |
+| L1 Weth               | [0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2](https://etherscan.io/address/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2) | [0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2](https://etherscan.io/address/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2)     | [0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6](https://goerli.etherscan.io/address/0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6)         |
+| L2 Weth               | [0x82aF49447D8a07e3bd95BD0d56f35241523fBab1](https://arbiscan.io/address/0x82aF49447D8a07e3bd95BD0d56f35241523fBab1)  | [0x722E8BdD2ce80A4422E880164f2079488e115365](https://nova.arbiscan.io/address/0x722E8BdD2ce80A4422E880164f2079488e115365) | [0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3](https://goerli.arbiscan.arbitrum.io/address/0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3) |
 
 </em>
 #### Third party gateways
```

Comment:
need a line break before this line
the header is not working

https://nitro-docs-git-remove-rinkarby-offchain-labs.vercel.app/useful-addresses#token-bridge